urllib3/contrib/socks_proxy.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
```

Comment:
Hm normally I'd suggest `socks_.py` for internal stuff but that would be annoying for people importing.

How about some `absolute_import` action?